src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_Synthesized.vb

```diff
@@ -235,6 +235,112 @@ End Class
                                                     End Sub)
         End Sub
 
+        <Theory>
+        <MemberData(NameOf(OptimizationLevelTheoryData))>
+        Public Sub BaseMethodWrapper_DoNotInheritAttributes(optimizationLevel As OptimizationLevel)
+            Dim source =
+<compilation>
+    <file><![CDATA[
+Imports System
+Imports System.Threading.Tasks
+
+Public Class Attr
+    Inherits Attribute
+End Class
+
+Public Class C
+    <Attr>
+    Public Overridable Async Function GetIntAsync() As <Attr> Task(Of Integer)
+        Return 0
+    End Function
+End Class
+
+Public Class D
+    Inherits C
+
+    <Attr>
+    Public Overrides Async Function GetIntAsync() As <Attr> Task(Of Integer)
+        Return Await MyBase.GetIntAsync()
+    End Function
+End Class
+]]></file>
+</compilation>
+
+            Dim comp = CreateCompilationWithMscorlib45AndVBRuntime(source, options:=
+                                                     New VisualBasicCompilationOptions(OutputKind.DynamicallyLinkedLibrary).
+                                                     WithOptimizationLevel(optimizationLevel).
+                                                     WithMetadataImportOptions(MetadataImportOptions.All))
+
+            CompileAndVerify(comp, symbolValidator:=Sub(m As ModuleSymbol)
+                                                        Dim baseWrapper = m.ContainingAssembly.GetTypeByMetadataName("D").GetMethod("$VB$ClosureStub_GetIntAsync_MyBase")
+
+                                                        Dim expectedNames = If(optimizationLevel = OptimizationLevel.Release,
+                                                                               {"CompilerGeneratedAttribute", "CompilerGeneratedAttribute"},
+                                                                               {"CompilerGeneratedAttribute", "CompilerGeneratedAttribute", "DebuggerHiddenAttribute"})
+
+                                                        AssertEx.SetEqual(expectedNames, GetAttributeNames(baseWrapper.GetAttributes()))
+                                                        AssertEx.SetEqual({}, GetAttributeNames(baseWrapper.GetReturnTypeAttributes()))
```

Comment:
>SetEqual [](start = 65, length = 8)

Consider using ```Assert.Empty``` here and in the test below.

Reply:
And there is no reason to call GetAttributeNames when we expect no attributes.

---
In reply to: [347041733](https://github.com/dotnet/roslyn/pull/39830#discussion_r347041733) [](ancestors = 347041733)